### PR TITLE
add collaboration README

### PR DESCRIPTION
**Pull request by Overlrd**
This commit fills the colaboration/README.md file with the group's norms we put together, please review it quickly so we can merge it.

### CODE REVIEW

File: collaboration/README.md [hunk 1]
```diff
@@ -1,5 +1,93 @@
 # Collaboration
```
Comment (kamaraak) [inline collaboration/README.md:1]: This is a great job. 
Can you please replace heading 1 "Collaboration" with our group name "Team Nexus".

File: collaboration/README.md [hunk 2]
```diff
@@ -1,5 +1,93 @@
...
 
-<!-- group norms summary -->
+## Group Name: Team Nexus
```
Comment (kamaraak) [inline collaboration/README.md:3]: We can get rid of heading 2 ##Group name: .... and keep heading 1

**Comment by kamaraak** [comment]
##Great job Augustine 
Please see the review

File: collaboration/README.md (hunk 2 above)
Comment (Overlrd) [reply collaboration/README.md]: It's done, please review the new commit

Comment (kamaraak) [reply collaboration/README.md]: Great merging right away💯

Comment (Overlrd) [reply collaboration/README.md]: And also I noticed a change in the collaboration/README.md file on the main branch, it wasn't necessary, the content would have been merged from this pull request anyway. The change added on the main branch is causing a merge conflict to happen. It's just an empty line, couldn't be easier to solve it but let's do the things the right way, NO DIRECT CHANGES TO THE MAIN BRANCH. Looks like you have the rights to do changes, please check the [repo setup guide ](https://github.com/Overlrd/ET6-foundations-group-08/blob/main/collaboration/guide/0_repository_setup.md) and make sure you set the rules described inside, I you need some help I to set the rules, I will help.

Comment (kamaraak) [reply collaboration/README.md]: I have reached out to Evan on our current situation of adding you to the team
Once that's done you will gain full access to to the group repo

Comment (kamaraak) [reply collaboration/README.md]: I think I managed to resolve the conflict and have managed to merge your pull request into the main branch

**Comment by kamaraak** [comment]
Merged successfully and issue closed. I think we are doing fine

**Review by nasratmn — COMMENTED** [review]
This looks great we can finalize and merge it

**Pull request merged**